front/impact.php

```diff
@@ -40,9 +40,15 @@
 if (!empty($itemtype) && !empty($items_id) && Impact::assetExist($itemtype, $items_id)) {
    $item = new $itemtype;
    $item->getFromDB($items_id);
+
    Impact::loadLibs();
+   $graph = Impact::makeDataForCytoscape(Impact::buildGraph($item));
+   $params = Impact::prepareParams($item);
+
+   echo '<div id="impact_graph_view">';
```

Comment:
Does not `Impcat::displayGraphView()` does the same?

Reply:
Fixed in https://github.com/glpi-project/glpi/pull/6441/commits/2378c31b317d6b97fb3be66133e964acea9add75.